src/debugger/android/androidPlatform.ts

```diff
@@ -19,17 +19,21 @@ import {Package} from "../../common/node/package";
 export class AndroidPlatform implements IAppPlatform {
     private extensionMessageSender: ExtensionMessageSender;
 
+    private static MULTIPLE_DEVICES_ERROR = "error: more than one device/emulator";
```

Comment:
If the build fails, we don't continue, regardless of the error.
If the build succeeds **and** we have multiple emulators, we then continue and pick our target.